mne/transforms.py

```diff
@@ -1694,31 +1693,27 @@ def _compute_volume_registration(moving, static, pipeline, zooms, niter):
                 logger.info(f'Using original zooms for {step} ...')
             static_zoomed, static_affine = _reslice_normalize(
                 static, zooms[step])
-        # must be resliced every time because it is adjusted at every step
-        moving_zoomed, moving_affine = _reslice_normalize(moving, zooms[step])
+            moving_zoomed, moving_affine = _reslice_normalize(
+                moving, zooms[step])
         logger.info(f'Optimizing {step}:')
         if step == 'sdr':  # happens last
+            affine_map = AffineMap(reg_affine,  # apply registration here
+                                   static_zoomed.shape, static_affine,
+                                   moving_zoomed.shape, moving_affine)
+            moving_zoomed = affine_map.transform(moving_zoomed)
             sdr = imwarp.SymmetricDiffeomorphicRegistration(
                 metrics.CCMetric(3), niter[step])
             with wrapped_stdout(indent='    ', cull_newlines=True):
                 sdr_morph = sdr.optimize(static_zoomed, moving_zoomed,
-                                         static_affine, moving_affine)
-            moving_zoomed = sdr_morph.transform(moving_zoomed)
+                                         static_affine, static_affine)
```

Comment:
This was the trouble, because it was aligned, the affines should both be the static affine

Reply:
Indeed that makes sense!